apps/reflect/src/components/app/CommandList.tsx

```diff
@@ -57,23 +57,25 @@ const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
 
 const CommandList = () => {
   const commandList = trpc.command.getCommandList.useQuery();
-  const [isLoading, setIsLoading] = React.useState(true);
-  const [botCommands, setBotCommands] = React.useState<IBotCommand[]>();
-
-  const [expanded, setExpanded] = React.useState<string | false>(false);
+  const commandAlias = trpc.command.getAliasList.useQuery();
+  const [isLoading, setIsLoading] = useState(true);
+  const [botCommands, setBotCommands] = useState<IBotCommand[]>();
+  const [aliasCommands, setAliasCommands] = useState<any>();
```

Comment:
We should not use `any` type here.

Reply:
when we define IBotCommandAlias[] as a type the code throw error of not assignable type so its the reason why ı give any type

Reply:
i got it :)